compiler/qsc_linter/Cargo.toml

```
@@ -0,0 +1,25 @@
+[package]
+name = "qsc_linter"
+
+version.workspace = true
+authors.workspace = true
+edition.workspace = true
+homepage.workspace = true
+license.workspace = true
+repository.workspace = true
+
+[dependencies]
+miette = { workspace = true }
+qsc_ast = { path = "../qsc_ast" }
+qsc_hir = { path = "../qsc_hir" }
+qsc_data_structures = { path = "../qsc_data_structures" }
+serde = { workspace = true }
+thiserror = { workspace = true }
+
+[dev-dependencies]
+expect-test = { workspace = true }
+qsc_parse = { path = "../qsc_parse" }
+serde_json = { workspace = true }
+
+[lib]
+doctest = false
```

I know I had just told you to enable lints for the project with explicit clippy module attributes in the lib.rs, but as of https://github.com/microsoft/qsharp/pull/1224 you can now just inherit the lint settings from the workspace by adding a few lines here:

```suggestion
[lints]
workspace = true

[lib]
doctest = false
```